server/src/auth-router.ts

```diff
@@ -48,15 +51,25 @@ router.use(
 router.use(passport.initialize());
 router.use(passport.session());
 
-passport.serializeUser((user: any, done: Function) => done(null, user));
-passport.deserializeUser((sessionUser: any, done: Function) =>
-  done(null, sessionUser)
-);
+passport.serializeUser((user, done) => done(null, user));
+passport.deserializeUser((sessionUser, done) => done(null, sessionUser));
 
-if (DOMAIN) {
-  Auth0Strategy.prototype.authorizationParams = function (options: any) {
-    var options = options || {};
+const ENVIRONMENT_URLS: { [key: string]: string } = {
+  stage: 'https://commonvoice.allizom.org',
+  prod: 'https://commonvoice.mozilla.org',
+  dev: 'https://dev.voice.mozit.cloud',
+  sandbox: 'https://sandbox.voice.mozit.cloud',
+};
 
+if (DOMAIN) {
+  /*
+    https://github.com/auth0/passport-auth0/issues/53#issuecomment-379463280
+
+    TODO: Explore if we can remove this patch now:
+    https://github.com/auth0/passport-auth0/issues/53#issuecomment-728830811
+  */
+  // @ts-ignore: Property 'authorizationParams' does not exist on type 'StrategyInternal'.
```

Comment:
We want to avoid using ts-ignore at all costs really, here I'm using it cause theres a TODO to explore doing a better fix.